### DISCUSSION TITLE
After configuration of imbo from Apache, XML is returned and i am not sure what to do next

### DISCUSSION THREAD

**Discussion by mukul-123** [imbo/imbo#646] [Q&A]
Hi,
I have installed and configured imbo locally on my system but when i visit on URL i see the XML data and nothing else that how do i proceed further.
![imbo](https://github.com/imbo/imbo/assets/40534324/e15ac430-8962-4e1b-9ce4-60686da1cd62)

**Comment by christeredvartsen** [comment]
I suggest you have a look at the [documentation](https://imbo.readthedocs.io/en/latest/) to learn more about the features that Imbo provides.
